tests/sharing/lib/sharing.rb

```diff
@@ -1,5 +1,5 @@
 class Sharing
-  attr_accessor :couch_id, :couch_rev, :rules
+  attr_accessor :couch_id, :couch_rev, :rules, :members
```

Comment:
We should remove `rules` and `members` from the line below `attr_reader` (`attr_accessor` is shortcut for `attr_reader` + `attr_writer`)